Schools/_posts/2018-09-25-AEPSHEP.md

```diff
@@ -1,4 +1,12 @@
 ---
+<<<<<<< HEAD
```

Comment:
Something went really wrong here - this is a conflict marker...

Reply:
I have fixed the conflict.